packages/base/src/FormatSettings.js

```diff
@@ -7,7 +7,7 @@ const getFormatLocale = () => {
 	const fallback = () => {
 		let oLocale = getLocale();
 		// if any user settings have been defined, add the private use subtag "sapufmt"
-		if (!Object.keys(mSettings).length === 0) {
+		if (Object.keys(mSettings).length === 0) {
```

Comment:
Please try to find out what the idea here is... I believe this was meant to be:
if (!(Object.keys(mSettings).length === 0)) {
but can't really be sure......